src/zabapgit_definitions.prog.abap

```diff
@@ -183,60 +189,61 @@ CONSTANTS: gc_newline TYPE abap_char1 VALUE cl_abap_char_utilities=>newline.
 
 CONSTANTS: gc_english TYPE spras VALUE 'E'.
 
-CONSTANTS: gc_root_dir    TYPE string VALUE '/',
-           gc_dot_abapgit TYPE string VALUE '.abapgit.xml' ##NO_TEXT,
-           gc_author_regex TYPE string VALUE '^([\w\s\.@\-_1-9]+) <(.*)> (\d{10}) .\d{4}$' ##NO_TEXT.
+CONSTANTS: gc_root_dir     TYPE string VALUE '/',
+           gc_dot_abapgit  TYPE string VALUE '.abapgit.xml' ##NO_TEXT,
+           gc_author_regex TYPE string VALUE '^([\w\s\.@\-_1-9]+) <(.*)> (\d{10}).\d{4}$' ##NO_TEXT.
```

Comment:
Wow. Nice eye. But that is really bizarre. In my system it is correct! I'll need to adjust it manually in the branch I think